### PR TITLE
Add utility to combine docstrings of callables

### PR DESCRIPTION
**Pull request by abergou**
Add a function decorator that programmatically combines function docstrings from specified functions with that of the wrapped function.

### CODE REVIEW

**Comment by abergou** [comment]
@rr- do you perchance have time to review? Thanks!

File: docstring_parser/util.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        params = dict(
+            ChainMap(
+                *(
+                    {param.arg_name: param for param in doc.params}
+                    for doc in docs
+                )
+            )
+        )
+
+        for doc in docs[::-1]:
```
Comment (rr-) [inline docstring_parser/util.py:103]: Consider using `reversed(docs)`

File: docstring_parser/util.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            if not doc.short_description:
+                continue
+            comb_doc.short_description = doc.short_description
+            comb_doc.blank_after_short_description = (
+                doc.blank_after_short_description
+            )
+            break
+        for doc in docs[::-1]:
```
Comment (rr-) [inline docstring_parser/util.py:111]: Consider using `reversed(docs)`

Comment (rr-) [reply docstring_parser/util.py:111]: Let's put a blank line before the `for` loop

File: docstring_parser/util.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            if not doc.long_description:
+                continue
+            comb_doc.long_description = doc.long_description
+            comb_doc.blank_after_long_description = (
+                doc.blank_after_long_description
+            )
+            break
+        combined = {}
```
Comment (rr-) [inline docstring_parser/util.py:119]: Let's put a blank line before `combined = {}`

**Review by rr- — CHANGES_REQUESTED** [review]
The function looks good but it's missing tests.

File: docstring_parser/util.py (hunk 1 above)
Comment (abergou) [reply docstring_parser/util.py]: Fixed.

File: docstring_parser/util.py (hunk 2 above)
Comment (abergou) [reply docstring_parser/util.py]: Fixed.

File: docstring_parser/util.py (hunk 3 above)
Comment (abergou) [reply docstring_parser/util.py:121]: Fixed.

**Comment by rr-** [comment]
CI is failing

**Comment by abergou** [comment]
> CI is failing (formatting)

Fixed. Thanks!

**Review by rr- — APPROVED** [review]

**Comment by rr-** [comment]
Thanks. Released as 0.11

**Pull request merged**